app-modules/form/src/Filament/Resources/FormResource/Pages/ManageFormEmailAutoReply.php

```diff
@@ -92,7 +93,26 @@ public function form(FilamentForm $form): FilamentForm
                         TextInput::make('subject')
                             ->required(fn (Get $get) => $get('is_enabled'))
                             ->placeholder('Subject')
-                            ->columnSpanFull(),
+                            ->columnSpanFull()
+                            ->visible(! RefactorEngagementCampaignSubjectToJsonb::active()),
+                        /**
+                         * @todo Remove 'temp_subject' name once `RefactorEngagementCampaignSubjectToJsonb` is removed.
+                         */
+                        TiptapEditor::make(RefactorEngagementCampaignSubjectToJsonb::active() ? 'subject' : 'temp_subject')
+                            ->mergeTags([
+                                'student first name',
+                                'student last name',
+                                'student full name',
+                                'student email',
+                                'student preferred name',
```

Comment:
Due to a recent change in one of Dan's ticket, this should now be `recipient` instead of `student`.